addons/project/report/project_report.py

```diff
@@ -137,6 +137,7 @@ def _group_by(self):
     def _from(self):
         return f"""
                 project_task t
+                    LEFT JOIN task_tree tree ON tree.id = t.id
```

Comment:
Why do we use `LEFT JOIN` and not `INNER JOIN`, I guess `task_tree` will contain all tasks, no?

Reply:
Yes indeed, it's fixed